### DISCUSSION TITLE
Unable to access the GitHub Octernships page

### DISCUSSION THREAD

**Discussion by bonzonic** [community/community#50473] [GitHub Education]
Hi,

I am having a 500 server error when I try to access this page:
https://education.github.com/globalcampus/octernships

I managed to access it previously but could not now. I'm accessing it from Malaysia.
 
![image](https://user-images.githubusercontent.com/76901263/226289665-b3d2dadc-54b3-435a-9c5a-f3b17c522362.png)

Any help would be appreciated!

**Comment by saaxena** [comment]
try opening in different browser , or  switch to private tab 
hope , it helps.

**Reply by bonzonic** [reply]
Thanks for the answer,
Though, I was met with the "I do not meet the conditions to join GitHub Octernships page" this morning which I assumed they fixed the 500 server error.
